scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -83,6 +92,71 @@ def _build_arg_parser():
     return p
 
 
+def binarize_labels(atlas, label_list):
+    #Input atlas must be a numpy array
+    mask = np.zeros(atlas.shape, dtype=np.uint16)
+    for label in label_list:
+        bin = atlas == label
+        mask[bin] = 1
+
+    return mask
+
+
+def dilate_mask(mask, mask_shape, vox_size, radius):
+    is_to_dilate = mask == 1
+    is_background = mask == 0
+
+    # Get the list of indices
+    background_pos = np.argwhere(is_background) * vox_size
+    label_pos = np.argwhere(is_to_dilate) * vox_size
+    ckd_tree = cKDTree(label_pos)
+
+    # Compute the nearest labels for each voxel of the background
+    dist, indices = ckd_tree.query(
+        background_pos, k=1, distance_upper_bound=radius,
+        n_jobs=-1)
+
+    # Associate indices to the nearest label (in distance)
+    valid_nearest = np.squeeze(np.isfinite(dist))
+    id_background = np.flatnonzero(is_background)[valid_nearest]
+    id_label = np.flatnonzero(is_to_dilate)[indices[valid_nearest]]
+
+    # Change values of those background
+    mask = mask.flatten()
+    mask[id_background.T] = mask[id_label.T]
+    mask = mask.reshape(mask_shape)
+
+    return mask
+
+
+def save_outliers(out_path, dir_name, step, tracto_name, ext, outlier, sft):
+    new_path = os.path.join(out_path, dir_name)
+    if not os.path.exists(new_path):
+        os.mkdir(new_path)
+
+    save_tractogram(outlier, os.path.join(
+        new_path, tracto_name + "_" + dir_name + "_outliers" + ext))
+    save_tractogram(sft, os.path.join(
+        new_path, tracto_name + "_" + step + ext))
+    return
```

Comment:
No need for a `return` statement: the method will return automatically once its execution has finished.

Reply:
Done in dbd0b2a.